src/main/java/seedu/address/ui/EventCard.java

```diff
@@ -38,18 +43,59 @@ public class EventCard extends UiPart<Region> {
     @FXML
     private FlowPane attendees;
 
+    @FXML
+    private ImageView calendarIcon;
+
+    @FXML
+    private ImageView personIcon;
+
     /**
      * Creates a {@code EventCard} with the given {@code Event} and index to display.
      */
     public EventCard(Event event, int displayedIndex) {
         super(FXML);
         this.event = event;
         id.setText(displayedIndex + ". ");
-        name.setText(event.getEventName());
-        date.setText(event.getDate().toString());
-        event.getAttendees().stream()
+        name.setText("  " + event.getEventName());
+        date.setText("  " + event.getDate().format(DateTimeFormatter.ofPattern(" MMM dd, yyyy")));
+        List<Person> sortedAttendees = event.getAttendees().stream()
                 .sorted(Comparator.comparing(person -> person.getName().toString()))
-                .forEach(person ->
-                        attendees.getChildren().add(new Label(person.getName().toString())));
+                .toList();
+
+        int maxDisplay = 2;
+        int numOfAttendees = sortedAttendees.size();
+        attendees.getChildren().add(new Label("  "));
+        for (int i = 0; i < maxDisplay; i++) {
```

Comment:
This might be a source of errors - if `numOfAttendees` is 0, then `sortedAttendees.get(i)` might return null and lead to null pointer exception

Maybe we should check for `i < min(2, numOfAttendees)` instead?

Also, if there are no attendees, let's put "No attendees" for the string to display

Reply:
Good point, I'll change it and update the pr